package.json

```diff
@@ -10,100 +10,95 @@
         "url": "git+https://github.com/eyeseetea/dhis2-app-skeleton.git"
     },
     "dependencies": {
-        "@dhis2/app-runtime": "2.2.2",
-        "@dhis2/d2-i18n": "1.0.6",
+        "@dhis2/app-runtime": "2.8.0",
+        "@dhis2/d2-i18n": "1.1.0",
         "@dhis2/d2-i18n-extract": "1.0.8",
         "@dhis2/d2-i18n-generate": "1.2.0",
-        "@dhis2/d2-ui-core": "7.1.3",
-        "@dhis2/d2-ui-forms": "7.1.3",
-        "@dhis2/ui-core": "4.21.1",
-        "@dhis2/ui-widgets": "2.1.0",
-        "@material-ui/core": "4.11.2",
+        "@dhis2/ui": "6.9.0",
+        "@eyeseetea/d2-api": "1.8.5",
+        "@eyeseetea/d2-ui-components": "2.6.6",
+        "@eyeseetea/xlsx-populate": "4.1.0",
```

Comment:
Do we need xlsx-populate?

Reply:
Not really

Reply:
Feel free to clean them up, as much as you can 😄